tests/build/build.go

```diff
@@ -811,7 +811,7 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					GinkgoWriter.Printf("build status: %+v\n", buildStatus.PaC)
 
 					return buildStatus.PaC != nil && buildStatus.PaC.State == "error" && strings.Contains(buildStatus.PaC.ErrMessage, "Git repository is already handled by Pipelines as Code"), nil
-				}, time.Minute*1, time.Second*2).Should(BeTrue(), "build status is unexpected")
+				}, time.Minute*5, time.Second*2).Should(BeTrue(), "build status is unexpected")
```

Comment:
generally it should not take that longer than 1 minute, but when this issue occurs (not sure about the exact cause of delay),  looking at the logs of rhtapbugs-996, in components.json `go-component-hepe` component creation time `"creationTimestamp": "2023-11-21T15:57:25Z"` and from the build-service log, the component update at `2023-11-21T15:59:05.593Z`, taking more than 1 minute, I think we should try to increase it to 2 to 3 minutes. (5 minute is long time to wait)

```
2023-11-21T15:59:05.593Z        INFO    ComponentOnboarding     controllers/component_build_controller.go:489   updated component after build request: configure-pac    {"controller": "component", "controllerGroup": "appstudio.redhat.com", "controllerKind": "Component", "Component": {"name":"go-component-hepe","namespace":"build-e2e-ojpq-tenant"}, "namespace": "build-e2e-ojpq-tenant", "name": "go-component-hepe", "reconcileID": "aa484eec-2f5c-43e6-bf83-553f4c17e9ca", "ComponentGitSource": {"url":"https://github.com/redhat-appstudio-qe/sample-multi-component","revision":"multi-component-base-wwji","context":"go-component"}, "action": "UPDATE"}
```

Reply:
@tisutisu thank you. 
In [the last two failed jobs](https://search.ci.openshift.org/?search=cannot+unmarshal+build+status+from+component+annotation%3A+unexpected+end+of+JSON+input&maxAge=336h&context=1&type=build-log&name=.*appstudio-e2e-tests.*&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job), the component also updated less than 2 minutes after it was created. 
I decreased the timeout to 2 minutes.